### PR TITLE
[RA2 CH1-3] fix ra2 :refs:

### PR DESCRIPTION
**Pull request by rgstori**
changing all :doc: and :ref: to the same format:
```
:ref:`absolute/path:section/`
```
I didn't find any references in chapters 4 to the end

### CODE REVIEW

**Comment by rgstori** [comment]
may supersede #3009 and #3010

**Review by collivier — APPROVED** [review]
lgtm

File: doc/ref_arch/kubernetes/chapters/chapter01.rst [hunk 1]
```diff
@@ -7,7 +7,7 @@ Introduction
 The intention of this Reference Architecture is to develop a usable Kubernetes based platform for the Telecom operator
 community. The RA will be based on the standard Kubernetes platform where ever possible. This Reference Architecture
```
Comment (pgoyal01) [inline doc/ref_arch/kubernetes/chapters/chapter01.rst:8]:
```suggestion
community. The RA will be based on the standard Kubernetes platform wherever possible. This Reference Architecture
```

File: doc/ref_arch/kubernetes/chapters/chapter01.rst [hunk 2]
```diff
@@ -7,7 +7,7 @@ Introduction
...
 for Kubernetes will describe the high level system components and their interactions, taking the
-:doc:`common/chapter00:goals and requirements` and mapping them to real-world Kubernetes (and related)
+:ref:`common/chapter00:overview` and mapping them to real-world Kubernetes (and related)
```
Comment (pgoyal01) [inline doc/ref_arch/kubernetes/chapters/chapter01.rst:10]: If referring to the chapter and not a specific section then you don't need the ":overview"

File: doc/ref_arch/kubernetes/chapters/chapter01.rst [hunk 3]
```diff
@@ -41,12 +41,12 @@ Kubernetes-based workloads, and Kubernetes cluster lifecycle management.
 Terminology
 ~~~~~~~~~~~
 
-For terminology used in this document refer to the :doc:`common/glossary:glossary`.
+For terminology used in this document refer to the :ref:`common/glossary:glossary`.
```
Comment (pgoyal01) [inline doc/ref_arch/kubernetes/chapters/chapter01.rst:44]:
```suggestion
For terminology used in this document refer to the :ref:`common/glossary`.
```

File: doc/ref_arch/kubernetes/chapters/chapter03.rst [hunk 4]
```diff
@@ -576,8 +574,8 @@ traffic require the capability to use a user space networking technology.
    Editors note: The possibility to SR-IOV for DPDK is under discussion.
 
    Refer to software profile features
-   :doc:`ref_model/chapters/chapter05:here ` and hardware profile
-   features :doc:`ref_model/chapters/chapter05:here `.
+   :doc:`ref_model/chapters/chapter05` and hardware profile
```
Comment (pgoyal01) [inline doc/ref_arch/kubernetes/chapters/chapter03.rst:577]:
```suggestion
   :doc:`ref_model/chapters/chapter05:#cloud infrastructure software profiles features and requirements` and hardware profile
```

File: doc/ref_arch/kubernetes/chapters/chapter03.rst [hunk 5]
```diff
@@ -576,8 +574,8 @@ traffic require the capability to use a user space networking technology.
...
+   features :doc:`ref_model/chapters/chapter05`.
```
Comment (pgoyal01) [inline doc/ref_arch/kubernetes/chapters/chapter03.rst:578]:
```suggestion
   features :doc:`ref_model/chapters/chapter05:#cloud infrastructure hardware profiles features and requirements`.
```

File: doc/ref_arch/kubernetes/chapters/chapter01.rst (hunk 3 above)
Comment (rgstori) [reply doc/ref_arch/kubernetes/chapters/chapter01.rst:44]: for some reason
```
:ref:`common/glossary`
```
(without `:section`) doesn't build, but
```
:doc:`common/glossary`
```
does

File: doc/ref_arch/kubernetes/chapters/chapter01.rst (hunk 2 above)
Comment (rgstori) [reply doc/ref_arch/kubernetes/chapters/chapter01.rst]: should be fixed now, replaced with doc

File: doc/ref_arch/kubernetes/chapters/chapter03.rst (hunk 5 above)
Comment (rgstori) [reply doc/ref_arch/kubernetes/chapters/chapter03.rst]: changed from doc to ref

File: doc/ref_arch/kubernetes/chapters/chapter01.rst [hunk 6]
```diff
@@ -41,12 +41,12 @@ Kubernetes-based workloads, and Kubernetes cluster lifecycle management.
 Terminology
 ~~~~~~~~~~~
 
-For terminology used in this document refer to the :doc:`common/glossary:glossary`.
+For terminology used in this document refer to the :doc:`common/glossary`.
```
Comment (CsatariGergely) [inline doc/ref_arch/kubernetes/chapters/chapter01.rst:44]:
```suggestion
For terminology used in this document refer to the :ref:`common/glossary:glossary`.
```

File: doc/ref_arch/kubernetes/chapters/chapter01.rst [hunk 7]
```diff
@@ -95,13 +95,13 @@ are many similarities, but the key principles from both, which are applicable to
...
-Anuket specifications define certain policies and :doc:`common/chapter00:principles` and strive to
+Anuket specifications define certain policies and :ref:`common/chapter00:anuket general principles` and strive to
 coalesce the industry towards conformant Cloud Infrastructure technologies and configurations. With the currently
 available technology options, incompatibilities, performance and operator constraints (including costs), these
 policies and principles may not always be achievable and, thus, require an exception process. These policies
-describe how to handle :doc:`common/policies:non-conforming technologies`.
+describe how to handle :ref:`common/policies:anuket project policies for managing non-conforming technologies`.
 In general, non-conformance with policies is handled through a set of exceptions (please also see
-:doc:`gov/chapters/chapter09:Exception Types`.
+:ref:`gov/chapters/chapter09:exception types`.
```
Comment (CsatariGergely) [inline doc/ref_arch/kubernetes/chapters/chapter01.rst:104]:
```suggestion
:ref:`gov/chapters/chapter09:exception types`).
```

**Review by CsatariGergely — CHANGES_REQUESTED** [review]

**Pull request merged**